velox/tpcds/gen/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_policy(SET CMP0079 NEW)
+
+project(TPCDS)
+
+# Add subdirectories
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/dsdgen build)
+
+add_library(append_info OBJECT utils/append_info-c.cpp)
```

Comment:
Do we need `velox_add_library` in this file @assignUser ?

Reply:
Jacob and I discussed this offline. We need to use velox_add_library here as well.

Reply:
@majetideepak, reverted the `velox_add_library` changes as they were resulting in a CI error when used with `-w` compile option:
```
CMake Error at velox/connectors/tpcds/CMakeLists.txt:16 (target_compile_options):
  target_compile_options can not be used on an ALIAS target.
```
Could you please take another look?